src/services/balancer/gauges/gauges.decorator.ts

```diff
@@ -153,6 +164,14 @@ export class GaugesDecorator {
   private resetMulticaller(): Multicaller {
     return new Multicaller(this.config.network.key, this.provider, this.abi);
   }
+
+  private resetHelperMulticaller(): Multicaller {
+    return new Multicaller(
+      this.config.network.key,
+      this.provider,
+      this.rewardsHelperAbi
+    );
+  }
```

Comment:
I'd just update `resetMulticaller` to take the abi as a param and then update usage in this class.
```suggestion
  private resetMulticaller(abi): Multicaller {
    return new Multicaller(this.config.network.key, this.provider, abi);
  }
```